### ISSUE TITLE
Error Running Idleon Saver

### ISSUE THREAD

**Issue by Sotales** [desophos/idleon-saver#11]
**Describe the problem**
Idleon Saver reports the following: "Something went wrong. Make sure Steam and Idleon is closed. Make sure to have recently played a character."

**To Reproduce**
Steps to reproduce the behavior, 
*Steam is running.
*Logged into Idleon and each character claimed items. Returned to Player Menu and exited the game.
*Loaded Idleon Saver and navigated to correct Idleon path as I have it in a custom directory. Idleon Saver is in same folder as Idleon.exe.
*Idleon Saver launches Idleon, but reports error as above.

**Expected behavior**
File created/outputted as expected.

**Log file**
Please attach your log file, found at `%APPDATA%\IdleonSaver\idleon_saver.log`.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10

**Additional context**
Idleon and Steam are in non-default locations.
[log_21-10-31_2.txt](https://github.com/desophos/idleon-saver/files/7450074/log_21-10-31_2.txt)

![Screenshot 2021-10-31 193955](https://user-images.githubusercontent.com/23343557/139607246-29fcd2ee-89f5-4538-85c3-1f8fe6432864.png)

**Comment by Sotales** [comment]
Manually inputting the directory location worked. Would not work if I used the browse function of Idleon Saver.

**Comment by desophos** [comment]
Do you mean that when you manually input the path, you were able to export your data without an error? What's the actual path to your LegendsOfIdleon.exe?

**Comment by Sotales** [comment]
Right, when I navigate with the browse button in Idleon Saver, it doesn't work. But when I copy and paste the path from the right directory, it works. Both attempt to open Idleon, so I'm not sure why it works one way and not the other. Attached are screen shots of the path used. First the one that is found via using the browse button, the second is the copy-pasted one. 
![wont work](https://user-images.githubusercontent.com/23343557/139693379-b175d3e8-5e54-4780-b2c5-4263f3662339.png)
![works](https://user-images.githubusercontent.com/23343557/139693392-c0ec333c-e70d-4f35-8c7a-9a5daff57bbf.png)

.

**Comment by desophos** [comment]
Those screenshots are very helpful, thank you! Could you post a screenshot of your Idleon install path in Windows File Explorer please?
![idleon path](https://user-images.githubusercontent.com/1887450/139726692-05b2c700-5482-4729-8633-90bdf172c040.png)

**Comment by Sotales** [comment]
![image](https://user-images.githubusercontent.com/23343557/139728122-86226289-bccc-4217-bb0f-387dcf26cb1b.png)

**Comment by desophos** [comment]
Thanks! It seems like Kivy's FileChooser is capitalizing your steamapps folder wrong for some reason, which is really strange. Could you post a screenshot of your Steam folder showing the name of your steamapps folder in Idleon Saver's Browse window, please? I'd like to see how it's displayed. Do you remember if your steamapps folder was ever named SteamApps?

**Comment by Sotales** [comment]
![image](https://user-images.githubusercontent.com/23343557/139729228-1f909fc2-338d-4529-ab6b-fcdfa146d3ff.png)


Don't think I ever renamed it. Just used Steam to create a new library location, I think.

**Comment by desophos** [comment]
Wow, I guess this is a Kivy bug! Thanks for bearing with me. I'll file an issue with Kivy if it doesn't already exist, but until then, you'll have to manually enter the path as a workaround (or just change "SteamApps" to "steamapps", which seems like the only problem). Sorry! Out of curiosity, is the folder still called "SteamApps" in the List View?

**Comment by Sotales** [comment]
![image](https://user-images.githubusercontent.com/23343557/139729970-93265af4-978b-4c4c-ba3c-90be02d2d4a0.png)

**Comment by Sotales** [comment]
It's capitalized in list view. So odd.

**Comment by desophos** [comment]
Thanks for confirming!

**Comment by Sotales** [comment]
![image](https://user-images.githubusercontent.com/23343557/139730270-4148ade4-4cca-4e7d-ad25-9df780ee84a0.png)

Yep. Hope this all helps.

**Comment by mnor42** [comment]
Seems that the app wont lookup any other HDD other than the C:/ 
Can you confirm ?
D:\Games\steamlibrary\steamapps\common\legendsofidleon.exe
